src/Api/Model/Languageforge/Lexicon/Command/LexUploadCommands.php

```diff
@@ -109,29 +109,21 @@ public static function uploadAudioFile($projectId, $mediaType, $tmpFilePath)
                 $ffprobeCommand = `ffprobe -i $sanitizedTmpFilePath -show_entries format=duration -v quiet -of csv="p=0" 2> /dev/null`;
```

Comment:
Per our discussion in person, we decided upon inspection of the variable that $tmpFilePath does not contain user input and is given to us by the calling method in the controller with a unique id.  So there is no need to sanitize the variable.  Thought I would note this for @longrunningprocess since he and I had flagged it earlier.

Let's remove the sanitize parts all together to make this simpler.

Reply:
The sanitized variable has been removed